README.md

```diff
@@ -132,6 +132,10 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### 2022-04-25
+
+* Rename AwardCriterionNumber to `CriterionNumber` and update descriptions in "criterionThreshold.csv"
```

Comment:
Based on the [bids extension changelog](https://extensions.open-contracting.org/en/extensions/bids/master/#changelog), codelist CSV names should be in backticks:

```suggestion
* Rename AwardCriterionNumber to `CriterionNumber` and update descriptions in `criterionThreshold.csv`
```